### ISSUE TITLE
Checkout mutation working only every second time

### ISSUE THREAD

**Issue by kedvardas1** [wp-graphql/wp-graphql-woocommerce#614]
Help needed!
We are using two Checkout mutations. One is for successful order with isPaid: true. Second one with isPaid: false if stripe payment intent fails (we are using 3DS here in Europe). And the problem is with isPaid: false.

**Steps to reproduce the behavior:**
1. Order created in Woocommerce with status "Failed" after user fails a payment intent. He gets message from stripe elements that payment failed due something.
2. User clicks same button and same flow goes again. Anyway if payment is succesful in Stripe or not Woocommerce returns Payment processing error failed (error comes from Stripe plugin as I debugged but that's try catch from WP)

**Expected behavior**
I am sure it should not show "Payment processing failed." error. We should see thank you page if payment intent successful or error from stripe react elements if payment intent fails.

**Here is the mutation if payment fails from first time**
`mutation Checkout(
    $metaData: [MetaDataInput]
    $billing: CustomerAddressInput
  ) {
    checkout(
      input: {
        paymentMethod: "stripe"
        metaData: $metaData
        billing: $billing
        isPaid: false
      }
    ) {
      order {
        cartHash
        databaseId
        needsPayment
      }
    }
  }`

**Here is the mutation for success payment**
`mutation Checkout(
    $metaData: [MetaDataInput]
    $billing: CustomerAddressInput
    $transactionId: String!
  ) {
    checkout(
      input: {
        paymentMethod: "stripe"
        metaData: $metaData
        billing: $billing
        isPaid: true
        transactionId: $transactionId
      }
    ) {
      order {
        cartHash
        databaseId
        needsPayment
      }
    }
  }`

Plugin Versions**
- **WooGraphQL Version: 0.10.6** …
- **WPGraphQL Version: 1.6.5** …
- **WordPress Version: 5.8**
- **WooCommerce Version: 5.6.0
- Stripe 5.6.1

Any suggestions, documentation or problems already solved somewhere?

**Comment by kidunot89** [comment]
@kedvardas1 If you want to process a payment don't set `isPaid`.

`isPaid: true` tells WooGraphQL to skip the payment processing step and that this order was paid for outside of WooCommerce.

**Comment by kedvardas1** [comment]
@kidunot89 yes I know. we are using isPaid: true only when payment is succesful from React Stripe elements.

the problem I wrote about is with isPaid: false

**Comment by kidunot89** [comment]
@kedvardas1 passing false to `isPaid` has no purpose. If you aren't setting it to `true` leave it out.

**Comment by kidunot89** [comment]
@kedvardas1 are you passing the stripe intent as metaData?

**Comment by kedvardas1** [comment]
@kidunot89 okay, thanks. 

yes, as metaData. stripe intent id and customer id

**Comment by kidunot89** [comment]
@kedvardas1 Any movement on this?
